roles/edpm_nova/templates/config.json.j2

```diff
@@ -0,0 +1,41 @@
+{
+    "command": "nova-compute",
```

Comment:
doesn't this need to be updated so that it reads all config files from /etc/nova, or does that happen by default?

Reply:
Looks like no. It seems --config-dir /etc/nova/nova.conf.d is the default behavior, is that right?

Reply:
corret it does not.

this is part of oslo oslo will look in /etc/nova and /etc/nova/nova.conf.d by default.
when we register the config options we tell it the project name e.g. "nova"
it then searches a set of directors to check incluign /etc/<project> and /etc/<project>/<project>.conf.d/

so we don't need to pass it here provided we place the files in the default locations
we we do below.